dev_tools/pr_monitor/README.md

```diff
@@ -1,105 +1,105 @@
 # PR monitor bot
 
-The pr monitor bot continuously watches a github repository for PRs labelled with the
-'automerge' label. When it sees such a PR it will mark it by labelling it with
-the 'front_of_queue_automerge' label. While there is an 'front_of_queue_automerge'
-labelled PR, the script will not label any other PRs with 'front_of_queue_automerge'.
-If there are multiple 'automerge' PRs, the bot prefers PRs that require less work 
-(e.g. tests already passed) and then breaks ties using the lowest PR number.
-
-While there is a 'front_of_queue_automerge' labelled PR, depending on the state 
-of the PR, the script might do all of the following: 
- * sync that PR with main
- * wait for status checks to succeed
- * attempt to merge it into main.
-If the PR goes out of date due to an intervening merge, the process will start over.
-This will continue until either the PR is merged or there is a problem that must be
-addressed by a human. After merging, the PR will be deleted unless it belongs to a
-fork.
-
-The commit message used when merging a PR is the title of the PR and then, for details,
-the body of the PR's initial message/comment. Users/admins should edit the title and
-initial comment to appropriately describe the PR.
-
-This script will also automatically label PRs based on their code size. The following
-labels are given based on the total change numbers (addition + deletions on github)
-at the time the PR is opened (not updated as changes are made):
-Extra Small (XS): < 10 total changes.
-Small (S):        < 50 total changes.
-Medium (M):       < 250 total changes.
-Large (L):        < 1000 total changes.
-Extra Large (XL): >= 1000 total changes.
+The pr monitor bot continuously watches a github repository for PRs labelled
+with the 'automerge' label. When it sees such a PR it will mark it by labelling
+it with the 'front_of_queue_automerge' label. While there is an
+'front_of_queue_automerge' labelled PR, the script will not label any other PRs
+with 'front_of_queue_automerge'. If there are multiple 'automerge' PRs, the bot
+prefers PRs that require less work (e.g. tests already passed) and then breaks
+ties using the lowest PR number.
+
+While there is a 'front_of_queue_automerge' labelled PR, depending on the state
+of the PR, the script might do all of the following:
+
+*   sync that PR with main
```

Comment:
Is the change `* sync` --> `*   sync` required by the linter?

The GH markdown specifications suggest the initial text, which most of contributors would be likely to use.